example/ExampleDialect.td

```diff
@@ -141,6 +141,13 @@ def Add32Op : ExampleOp<"add32", [Memory<[]>, NoUnwind, WillReturn]> {
   let results = (outs I32:$result);
   let arguments = (ins I32:$lhs, I32:$rhs, AttrI32:$extra);
 
+  let value_traits = [
+    (NoCapture $lhs),
+    (NoCapture $rhs),
```

Comment:
Having `nocapture` on an int argument is pretty strange. Best to add that to an op that has pointer types.